packages/manager/apps/pci-rancher/src/hooks/useRancher.tsx

```diff
@@ -18,13 +18,13 @@ export const useRancher = ({
 } = {}) => {
   const { projectId, rancherId } = useParams();
   return useQuery<RancherService, ErrorResponse>({
-    queryKey: ['project', projectId, 'rancher', rancherId],
+    queryKey: ['projectId', projectId, 'rancher', rancherId],
```

Comment:
If `project` has been switched to `projectId`, I think we should also harmonize it with `rancher` ;)

Reply:
I switch to `projectId` to fix a conflict cache key from `useProject` hook. But yes I agree with you, I change it so to harmonize